### PR TITLE
Embedded CRT YouTube video on the page

### PR DESCRIPTION
**Pull request by cwarnermm**
Embedded the Collapsed Reply Threads (Beta) video on the documentation page.

### CODE REVIEW

File: source/help/messaging/organizing-conversations.rst [hunk 1]
```diff
@@ -68,4 +68,10 @@ Select **Unreads** to filter your followed threads by only those with unread rep
 Tutorial Video
 ---------------
 
-For additional details about working with the Beta release of Collapsed Reply Threads, see our `tutorial video <https://youtu.be/hnD0Zj-mIbs>`__. 
+For additional details about working with the Beta release of Collapsed Reply Threads, see our `tutorial video on YouTube <https://youtu.be/hnD0Zj-mIbs>`__.
```
Comment (cwarnermm) [inline source/help/messaging/organizing-conversations.rst:71]:
```suggestion
For additional details about working with the Beta release of Collapsed Reply Threads, see our `tutorial video on YouTube <www.youtube.com/embed/hnD0Zj-mIbs>`__. 
```

File: source/help/messaging/organizing-conversations.rst [hunk 2]
```diff
@@ -68,4 +68,10 @@ Select **Unreads** to filter your followed threads by only those with unread rep
 Tutorial Video
 ---------------
 
-For additional details about working with the Beta release of Collapsed Reply Threads, see our `tutorial video <https://youtu.be/hnD0Zj-mIbs>`__. 
+For additional details about working with the Beta release of Collapsed Reply Threads, see our `tutorial video on YouTube <https://youtu.be/hnD0Zj-mIbs>`__. 
+
+.. raw:: html
+
+   <div style="position: relative; padding-bottom: 50%; height: 0; overflow: hidden; max-width: 100%; height: auto;">
+      <iframe src="https://youtu.be/hnD0Zj-mIbs" frameborder="0" allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 95%;"></iframe>
```
Comment (cwarnermm) [inline source/help/messaging/organizing-conversations.rst:76]:
```suggestion
      <iframe src="www.youtube.com/embed/hnD0Zj-mIbs" frameborder="0" allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 95%;"></iframe>
```

File: source/help/messaging/organizing-conversations.rst [hunk 3]
```diff
@@ -68,4 +68,10 @@ Select **Unreads** to filter your followed threads by only those with unread rep
 Tutorial Video
 ---------------
 
-For additional details about working with the Beta release of Collapsed Reply Threads, see our `tutorial video <https://youtu.be/hnD0Zj-mIbs>`__. 
+For additional details about working with the Beta release of Collapsed Reply Threads, see our `tutorial video on YouTube <www.youtube.com/embed/hnD0Zj-mIbs>`__.
```
Comment (cwarnermm) [inline source/help/messaging/organizing-conversations.rst:71]:
```suggestion
For additional details about working with the Beta release of Collapsed Reply Threads, see our `tutorial video on YouTube <www.youtube.com/watch?v=hnD0Zj-mIbs>`_. 
```

Comment (cwarnermm) [inline source/help/messaging/organizing-conversations.rst]:
```suggestion
For additional details about working with the Beta release of Collapsed Reply Threads, see our `tutorial video on YouTube <https://www.youtube.com/embed/hnD0Zj-mIbs>`__. 
```

File: source/help/messaging/organizing-conversations.rst [hunk 4]
```diff
@@ -68,4 +68,10 @@ Select **Unreads** to filter your followed threads by only those with unread rep
 Tutorial Video
 ---------------
 
-For additional details about working with the Beta release of Collapsed Reply Threads, see our `tutorial video <https://youtu.be/hnD0Zj-mIbs>`__. 
+For additional details about working with the Beta release of Collapsed Reply Threads, see our `tutorial video on YouTube <www.youtube.com/embed/hnD0Zj-mIbs>`__. 
+
+.. raw:: html
+
+   <div style="position: relative; padding-bottom: 50%; height: 0; overflow: hidden; max-width: 100%; height: auto;">
+      <iframe src="www.youtube.com/embed/hnD0Zj-mIbs" frameborder="0" allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 95%;"></iframe>
```
Comment (cwarnermm) [inline source/help/messaging/organizing-conversations.rst:76]:
```suggestion
      <iframe src="https://www.youtube.com/embed/hnD0Zj-mIbs" frameborder="0" allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 95%;"></iframe>
```

File: source/help/messaging/organizing-conversations.rst [hunk 5]
```diff
@@ -68,4 +68,10 @@ Select **Unreads** to filter your followed threads by only those with unread rep
 Tutorial Video
 ---------------
 
-For additional details about working with the Beta release of Collapsed Reply Threads, see our `tutorial video <https://youtu.be/hnD0Zj-mIbs>`__. 
+For additional details about working with the Beta release of Collapsed Reply Threads, see our `tutorial video on YouTube <www.youtube.com/watch?v=hnD0Zj-mIbs>`_.
```
Comment (cwarnermm) [inline source/help/messaging/organizing-conversations.rst:71]:
```suggestion
For additional details about working with the Beta release of Collapsed Reply Threads, see our `tutorial video on YouTube <https://www.youtube.com/watch?v=hnD0Zj-mIbs>`_. 
```

**Review by justinegeffen — APPROVED** [review]
Absolutely fantastic! Yay!

**Pull request merged**